modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -1157,9 +1157,9 @@ return view.extend({
 						o.placeholder = '';
 
 					o = ss.taboption('advanced', form.Value, 'macaddr', _('MAC address'), _('Override default MAC address - the range of usable addresses might be limited by the driver'));
-					o.optional = true;
-					o.placeholder = radioNet.getActiveBSSID();
-					o.datatype = 'macaddr';
+					o.value('', _('automatic (no override)'));
```

Comment:
```suggestion
					o.value('', _('automatic (%s)').format(radioNet.getActiveBSSID()));
```

I'd prefer that you still see the address.

Reply:
I think this will display the randomly generated MAC address there if the current setting is set to random? It will be misleading.

Reply:
Do you know of a way to read the next available/assign-able MAC address for the actual WiFi interface from `sysfs` or some other place as appropriate?

Reply:
Implemented in https://github.com/openwrt/luci/pull/5896/commits/8cb87cbbf7dd4ff48a09d2e3d248cd7db6c21266